operations/mimir-mixin/dashboards/tenants.libsonnet

```diff
@@ -530,5 +530,83 @@ local filename = 'mimir-tenants.json';
         ) +
         { legend: { show: false } },
       )
-    ),
+    )
+
+    .addRow(
+      $.row('Read Path - Insights')
+      .addPanel(
+        local title = 'Rate of Read Requests';
+        $.panel(title) +
+        $.queryPanel(
+          'sum(rate(cortex_query_frontend_queries_total{%s, user="$user"}[$__rate_interval]))' % $.namespaceMatcher(),
+          'rate'
+        )
+      )
+      .addPanel(
+        local title = 'Number of Queries Queued';
+        $.panel(title) +
+        $.queryPanel(
+          [
+            'sum(cortex_query_scheduler_queue_length{%s, container="query-scheduler", user="$user"})' % $.namespaceMatcher(),
+          ],
+          [
+            'queue length',
+          ],
+        )
+      )
+    )
+
+    .addRow(
+      ($.row('Read Path - Queries') + { collapse: true })
+      .addPanel(
+        local title = 'Query Success vs Failure Rate';
+        $.panel(title) +
+        $.queryPanel(
+          [
+            'sum(rate({%s, name=~"query-frontend.*"} |= "msg=\\"query stats\\"" |= "user=$user" |= "status=success" [$__interval]))' % $.namespaceMatcher(),
+            'sum(rate({%s, name=~"query-frontend.*"} |= "msg=\\"query stats\\"" |= "user=$user" |= "status=failed" [$__interval]))' % $.namespaceMatcher(),
+          ],
+          [
+            'success rate',
+            'failure rate',
+          ],
+        ) + {
+          datasource: '${lokidatasource}',
+        }
+      )
+      .addPanel(
+        local title = 'Query Response Time';
+        $.panel(title) +
+        $.queryPanel(
+          [
+            'quantile_over_time(0.99, {%s ,name=~"query-frontend.*"}  |= "msg=\\"query stats\\"" |= "user=$user" | logfmt | unwrap duration(response_time)[$__interval]) by (user)' % $.namespaceMatcher(),
+            'quantile_over_time(0.50, {%s ,name=~"query-frontend.*"}  |= "msg=\\"query stats\\"" |= "user=$user" | logfmt | unwrap duration(response_time)[$__interval]) by (user)' % $.namespaceMatcher(),
+            'quantile_over_time(0.95, {%s ,name=~"query-frontend.*"}  |= "msg=\\"query stats\\"" |= "user=$user" | logfmt | unwrap duration(response_time)[$__interval]) by (user)' % $.namespaceMatcher(),
+          ],
+          [
+            'p99',
+            'p50',
+            'p95',
```

Comment:
For consistency with our latency panels, we could do:
- 99th Percentile
- 50th Percentile
- Average